templates/defaults.j2

```diff
@@ -1,5 +1,6 @@
 CONSUL_STARTOPS="{{ consul.startopts }}"
 BIND_INTERFACE="{{ consul.bind_interface | default('eth0') }}"
+BIND="$(ifconfig {{  consul.bind_interface | default('eth0') }} | grep "inet addr" | awk '{ print substr($2,6) }')"
```

Comment:
Can you reuse BIND_INTERFACE and update the upstart config appropriately? I think we should keep the two init methods aligned as much as possible.